messages/TzMsg_ja.properties

```diff
@@ -15,136 +15,137 @@
-Asia/Kathmandu = GMT +05:45 \u30ab\u30c8\u30de\u30f3\u30ba
-Asia/Colombo = GMT +05:30 \u30b9\u30ea\u30fb\u30b8\u30e3\u30e4\u30ef\u30eb\u30c0\u30ca\u30d7\u30e9\u30fb\u30b3\u30c3\u30c6
-Asia/Damascus = GMT +02:00 \u30c0\u30de\u30b9\u30ab\u30b9
-Asia/Dhaka = GMT +06:00 \u30c0\u30c3\u30ab
-Asia/Hong_Kong = GMT +08:00 \u5317\u4eac\u3001\u91cd\u6176\u3001\u9999\u6e2f\u3001\u30a6\u30eb\u30e0\u30c1
-Asia/Irkutsk = GMT +08:00 \u30a4\u30eb\u30af\u30fc\u30c4\u30af (RTZ 7)
-Asia/Jerusalem = GMT +02:00 \u30a8\u30eb\u30b5\u30ec\u30e0
-Asia/Kabul = GMT +04:30 \u30ab\u30d6\u30fc\u30eb
-Asia/Karachi = GMT +05:00 \u30a4\u30b9\u30e9\u30de\u30d0\u30fc\u30c9\u3001\u30ab\u30e9\u30c1
-Asia/Kathmandu = GMT +05:45 \u30ab\u30c8\u30de\u30f3\u30ba
-Asia/Krasnoyarsk = GMT +07:00 \u30af\u30e9\u30b9\u30ce\u30e4\u30eb\u30b9\u30af (RTZ 6)
-Asia/Kuala_Lumpur = GMT +08:00 \u30af\u30a2\u30e9\u30eb\u30f3\u30d7\u30fc\u30eb
-Asia/Kuwait = GMT +03:00 \u30af\u30a6\u30a7\u30fc\u30c8\u3001\u30ea\u30e4\u30c9
-Asia/Magadan = GMT +11:00 \u30de\u30ac\u30c0\u30f3
-Asia/Muscat = GMT +04:00 \u30a2\u30d6\u30c0\u30d3\u3001\u30de\u30b9\u30ab\u30c3\u30c8
-Asia/Novosibirsk = GMT +07:00 \u30ce\u30dc\u30b7\u30d3\u30eb\u30b9\u30af (RTZ 5)
-Asia/Yangon = GMT +06:30 \u30e4\u30f3\u30b4\u30f3
-Asia/Seoul = GMT +09:00 \u97d3\u56fd
-Asia/Taipei = GMT +08:00 \u53f0\u5317
-Asia/Tashkent = GMT +05:00 \u30bf\u30b7\u30b1\u30f3\u30c8
-Asia/Tbilisi = GMT +04:00 \u30c8\u30ea\u30d3\u30b7
-Asia/Tehran = GMT +03:30 \u30c6\u30d8\u30e9\u30f3
-Asia/Tokyo = GMT +09:00 \u65e5\u672c
-Asia/Ulaanbaatar = GMT +08:00 \u30a6\u30e9\u30f3\u30d0\u30fc\u30c8\u30eb
-Asia/Pyongyang = GMT +08:30 \u5e73\u58cc
-Asia/Vladivostok = GMT +10:00 \u30a6\u30e9\u30b8\u30aa\u30b9\u30c8\u30af (RTZ 9)
-Asia/Yakutsk = GMT +09:00 \u30e4\u30af\u30fc\u30c4\u30af (RTZ 8)
-Asia/Yekaterinburg = GMT +05:00 \u30a8\u30ab\u30c6\u30ea\u30f3\u30d6\u30eb\u30af (RTZ 4)
-Asia/Yerevan = GMT +04:00 \u30a8\u30ec\u30d0\u30f3
-Atlantic/Azores = GMT -01:00 \u30a2\u30be\u30ec\u30b9
-Atlantic/Cape_Verde = GMT -01:00 \u30ab\u30dd\u30d9\u30eb\u30c7
-Atlantic/South_Georgia = GMT -02:00 \u4e2d\u592e\u5927\u897f\u6d0b
-Australia/Adelaide = GMT +09:30 \u30a2\u30c7\u30ec\u30fc\u30c9
-Australia/Brisbane = GMT +10:00 \u30d6\u30ea\u30b9\u30d9\u30f3
-Australia/Darwin = GMT +09:30 \u30c0\u30fc\u30a6\u30a3\u30f3
-Australia/Hobart = GMT +10:00 \u30db\u30d0\u30fc\u30c8
-Australia/Perth = GMT +08:00 \u30d1\u30fc\u30b9
-Australia/Sydney = GMT +10:00 \u30ad\u30e3\u30f3\u30d9\u30e9\u3001\u30e1\u30eb\u30dc\u30eb\u30f3\u3001\u30b7\u30c9\u30cb\u30fc
-America/Sao_Paulo = GMT -03:00 \u30d6\u30e9\u30b8\u30ea\u30a2
-America/Halifax = GMT -04:00 \u5927\u897f\u6d0b\u6a19\u6e96\u6642\uff08\u30ab\u30ca\u30c0\uff09
-America/St_Johns = GMT -03:30 \u30cb\u30e5\u30fc\u30d5\u30a1\u30f3\u30c9\u30e9\u30f3\u30c9
-America/Regina = GMT -06:00 \u30b5\u30b9\u30ab\u30c1\u30e5\u30ef\u30f3
-Etc/GMT+12 = GMT -12:00 \u65e5\u4ed8\u5909\u66f4\u7dda
+Africa/Algiers = GMT+01:00\u897f\u4e2d\u592e\u30a2\u30d5\u30ea\u30ab
```

Comment:
is the spaces around offset removed by mistake?
GMT+01:00 should be GMT +01:00

Reply:
no, the space is left by mistake in America/Cancun
There should be no space. Correcting this now